From tilman@imap.cc  Thu Mar 13 14:50:13 2008
From: Tilman Schmidt <tilman@imap.cc>
Date: Thu, 13 Mar 2008 19:51:42 +0100 (CET)
Subject: USB: usb.h: reduce syslog clutter [v3]
To: Greg KH <greg@kroah.com>, <linux-usb@vger.kernel.org>
Cc: Roland Kletzing <devzero@web.de>
Message-ID: <20080313185142.459F411005B@xenon.ts.pxnet.com>


The the err() / info() / warn() macros in usb.h inserted __FILE__ at
the beginning of the message, which expands to the complete pathname
of the source file within the kernel tree, frequently taking up half
of an 80 character screen line before the actual message even begins.
Use the module name instead.

Signed-off-by: Tilman Schmidt <tilman@imap.cc>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb.h |   13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -1701,13 +1701,12 @@ extern void usb_unregister_notify(struct
 #define dbg(format, arg...) do {} while (0)
 #endif
 
-#define err(format, arg...) printk(KERN_ERR "%s: " format "\n" , \
-	__FILE__ , ## arg)
-#define info(format, arg...) printk(KERN_INFO "%s: " format "\n" , \
-	__FILE__ , ## arg)
-#define warn(format, arg...) printk(KERN_WARNING "%s: " format "\n" , \
-	__FILE__ , ## arg)
-
+#define err(format, arg...) printk(KERN_ERR KBUILD_MODNAME ": " \
+	format "\n" , ## arg)
+#define info(format, arg...) printk(KERN_INFO KBUILD_MODNAME ": " \
+	format "\n" , ## arg)
+#define warn(format, arg...) printk(KERN_WARNING KBUILD_MODNAME ": " \
+	format "\n" , ## arg)
 
 #endif  /* __KERNEL__ */
 
